### ISSUE TITLE
Sorting string lists

### ISSUE THREAD

**Issue by nicolasr75** [vaadin/vaadin-grid#1358]
### Description

Occasionally I want to use vaadin-grid for displaying simple string arrays due to the great sorting feature. The question is then how do I configure the sorting? vaadin-grid-sorter expects a path to a subproperty but in the case of a simple string array there is no subproperty. Leaving the `path` attribute empty does not work. 
Is this possible at all or do I have to wrap my array in an object array with a dummy subproperty?

**Comment by platosha** [comment]
Indeed, looks like empty sorter path is, unfortunately, not supported. This was not intentional, though, we should probably consider covering that.

In the meanwhile, as said, you can use an object array as a workaround. Another workaround option is to use a `dataProvider` callback instead of the `items` array, this would allow to implement any custom sorting.
